GVFS/GVFS.Common/FileSystem/PhysicalFileSystem.cs

```diff
@@ -27,9 +27,12 @@ public virtual void RecursiveDelete(string path)
                 file.Delete();

```

Comment:
We shouldn't be deleting the files inside `path` if `recursive` if false.  We should just be calling `directory.Delete();`.

Reply:
(We want to match the behavior of [Directory.Delete](https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.delete?view=netframework-4.7.2#System_IO_Directory_Delete_System_String_System_Boolean_))

Reply:
updated